libraries/shared/storage/utils/decoder.go

```diff
@@ -51,3 +60,52 @@ func decodeUint48(raw []byte) string {
 func decodeAddress(raw []byte) string {
 	return common.BytesToAddress(raw).Hex()
 }
+
+func decodePackedSlot(raw []byte, packedTypes map[int]ValueType) map[int]string {
+	storageSlotData := raw
```

Comment:
I think this assignment is superfluous; could name the argument `storageSlotData` instead?

Reply:
I assigned the `raw` argument to `storageSlotData` because then on line 84 we're updating what is in `storageSlotData` by popping off the current item. Does that make sense?

Reply:
Oh yeah that makes perfect sense, sorry I missed that!